content/blog/wingingit/winging-it-03.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Intuitive Web Design with OOUX
+episode: 3
+author:
```

Comment:
@mirisuzanne @jamesnw @jerivas tagging all of you since this is a bit time sensitive, but  when I added a tag I get an error in the terminal (see screenshot). 

```
tags:
  - ooux 
  ```

  also tried:

```  
  tags:
  - design
 ```

![Screenshot 2023-10-24 at 9 55 19 AM](https://github.com/oddbird/oddleventy/assets/1581694/8e2bcb8b-c5fc-4394-8837-91a6fbf857c6)